index.bs

```diff
@@ -4109,165 +4301,1226 @@ To <dfn>get the response data</dfn> given |response|:
+
+  1. Let |headers| be an empty [=/header list=].
+
+  1. For |header| in |command parameters|["<code>headers</code>"]:
+
+    1. Append [=deserialize header=] with |header| to |headers|.
+
+  1. Set |request|'s [=request/headers list=] to |headers|.
+
+1. If |command parameters| [=map/contains=] "<code>cookies</code>":
+
+  1. Let |cookie header| be an empty [=byte sequence=].
+
+  1. For each |cookie| in |command parameters|:
+
+    1. If |cookie header| is not empty, append `<code>;</code>` to |cookie
+       header|.
+
+    1. Append [=serialize cookie header=] with |cookie| to |cookie header|.
+
+  1. Let |found cookie header| be false.
+
+  1. For each |header| in |request|'s [=request/headers list=]:
+
+    1. Let |name| be |header|'s name.
+
+    1. If [=byte-lowercase=] |name| is `<code>cookie</code>`:
+
+      1. Set |header|'s value to |cookie header|.
+
+      1. Set |found cookie header| to true.
+
+      1. Break.
+
+  1. If |found cookie header| is false:
+
+    1. Append the [=header=] (`<code>Cookie</code>`, |cookie header|) to
+       |request|'s [=request/headers list=].
+
+1. If |command parameters| [=map/contains=] "<code>body</code>":
+
+  1. Let |body| be [=deserialize protocol bytes=] with |command
+     parameters|["<code>body</code>"].
+
+  1. Set |request|'s [=request/body=] to |body|.
+
+1. [=Resume=] with "<code>continue request</code>", |request id|, and (null,
+   "<code>incomplete</code>").
+
+1. Return [=success=] with [=data=] null.
+
+</div>
+
+#### The network.continueResponse Command ####  {#command-network-continueResponse}
+
+The <dfn export for=commands>network.continueResponse</dfn> command continues a
+response that's blocked by a [=network intercept=]. If called in the
+<code>beforeRequestSent</code> phase, this allows the request to be fulfilled
+without being sent. If called in the <code>responseStarted</code> phase, this
+allows the fetch response to be modified.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      network.ContinueResponse = {
+        method: "network.continueResponse",
+        params: network.ContinueResponseParameters
+      }
+
+      network.ContinueResponseParameters = {
+        request: network.Request,
+        ?cookies: [*network.SetCookieHeader]
+        ?credentials: network.AuthCredentials,
+        ?headers: [*network.Header],
+        ?reasonPhrase: text,
+        ?statusCode: js-uint,
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+To <dfn>update the response</dfn> given |session|, |command| and |command parameters|:
+
+1. Let |blocked requests| be |session|'s [=blocked request map=].
+
+1. Let |request id| be |command parameters|["<code>request</code>"].
+
+1. If |blocked requests| does not [=map/contain=] |request id| then return
+   [=error=] with [=error code=] [=invalid argument=].
```

Comment:
Regarding the issue below, `no such request`  is now an existing error.

Should `invalid argument` be replaced with `no such request` then?